frontend/src/metabase/query_builder/components/QuestionActions.tsx

```diff
@@ -0,0 +1,226 @@
+import React from "react";
+import { t } from "ttag";
+
+import { color } from "metabase/lib/colors";
+
+import Button from "metabase/core/components/Button";
+import Tooltip from "metabase/components/Tooltip";
```

Comment:
According to our [FE style guide](https://www.metabase.com/docs/latest/developers-guide/frontend.html#javascript-conventions), `metabase/lib` imports should come after `metabase/components` imports

Reply:
Unless enforced with a eslint rule, I wouldn't sort these imports manually